src/api/app/views/webui2/webui/repositories/index.html.haml

```diff
@@ -16,27 +16,48 @@
           This package is inherited from the project
           = link_to @package.project, action: :index, controller: :repositories, project: @package.project
           and changing repository flags has no effect.
-  - else
+- else
+  .card.mb-3
     - @pagetitle = "Repositories for #{@project}"
     = render partial: 'webui/project/tabs', locals: { project: @project }
     .card-body
-      %h3= @pagetitle
-      %p
-        You can configure individual flags for this project here.
+      %h3.pb-2= @pagetitle
 
-      #repository-list
-        - @project.repositories.includes(:path_elements, :download_repositories).each do |repository|
-          = render partial: 'repository_entry', locals: { repository: repository }
+      %ul.list-inline.mb-0
+        - if User.current.can_modify?(@project)
+          %li.list-inline-item
+            = link_to(repositories_distributions_path(project: @project), class: 'nav-link') do
+              %i.fas.fa-plus-circle.text-success
+              Add from a Distribution
 
-      - if User.current.is_admin?
-        .hidden.add-dod-repository-form
-          = render partial: 'dod_repository_form', locals: { project: @project, download_on_demand: DownloadRepository.new }
-        %p
-          = link_to(sprite_tag('drive_add', title: 'Add DoD repository') + ' Add DoD repository', '#', id: 'add-dod-repository-link')
-      - if User.current.can_modify?(@project)
-        %p
-          = link_to(sprite_tag('drive_add', title: 'Add repository') + ' Add repositories', action: :distributions, project: @project)
+          %li.list-inline-item
+            = link_to('#', data: { toggle: 'modal', target: '#add-repository-from-project' }, class: 'nav-link') do
+              %i.fas.fa-plus-circle.text-success
+              Add from a Project
+            = render partial: 'add_repository_from_project_modal', locals: { project: @project }
+
+        - if User.current.is_admin?
+          %li.list-inline-item
+            = link_to('#', data: { toggle: 'modal', target: '#add-dod-repository-modal' }, class: 'nav-link') do
+              %i.fas.fa-plus-circle.text-success
+              Add DoD Repository
+            =# render partial: 'add_dod_repository_modal', locals: { project: @project }
 
+  .card.mb-3
+    .card-header.d-flex.justify-content-between
+      %h5 My Repositories
```

Comment:
```suggestion
      %h5
        Repositories for #{@project}
```

Reply:
Ouch I just saw the comment :see_no_evil: I'll change it in the following PR :wink: